.github/CODEOWNERS

```diff
@@ -1,3 +1,5 @@
 # See https://docs.github.com/en/enterprise-server@2.21/github/creating-cloning-and-archiving-repositories/creating-a-repository-on-github/about-code-owners#example-of-a-codeowners-file
 
 *       christian@parpart.family
+/.github/ @data-man
+cmake/ @data-man
```

Comment:
Ugh. How doesit work again? Can there be more than one? I forgot what the benefit of code owners on GitHub was (sorry).

Reply:
You can delete this file if you trust me. :)

Reply:
The big example:
https://github.com/dlang/dmd/blob/master/CODEOWNERS

Reply:
What do you think about a contributors file on top of that?

Reply:
I don't know.
https://github.com/contour-terminal/contour/graphs/contributors is enough for me. :)

Reply:
I think the "Credits" section in the readme is more important as we use many third-party libraries.

Reply:
this PR is still in Draft mode for a reason?

Reply:
No, there is no specific reason. Just waiting for your final decision. :)